### ISSUE TITLE
Does not see libraries dependencies

### ISSUE THREAD

**Issue by ovitente** [rwyattwalker/dots#1]
_First of all thank you for that great configs set, I am new at awesome, but worked many years on i3._

OS: ubuntu 22.04
Awesome version: 
```
awesome v4.3 (Too long)
 • Compiled against Lua 5.3.6 (running with Lua 5.3)
 • D-Bus support: ✔
 • execinfo support: ✔
 • xcb-randr version: 1.6
 • LGI version: 0.9.2
```
The same as on git now.

I've just followed the readme and faced this 
![image](https://user-images.githubusercontent.com/10868371/200949983-552c483e-8bf8-43ce-888c-60001b6eacb3.png)
Through reading, I've figured out that it does not see lua files and expects it on the system directory. 
Putting them there did not help.

And directory `awesome-buttons` is empty in this dots repo.

Could you please update the readme to cover placing for these libs?

**Comment by rwyattwalker** [comment]
Added a step to the install process. I forgot to include installing the awesome-buttons repo. That's in the readme now and that should fix it. I am going to close this issue for now and if you still have issues please reopen it

**Comment by ovitente** [comment]
Thank you.
I followed new step, but now after log in I see only black window with cursor, I thought it is because I used default apt picom, so I've build it from sources from  https://github.com/jonaburg/picom, but result is the same.

Then I commented picom autostart in rc.lua config and re-launched session.

![image](https://user-images.githubusercontent.com/10868371/201079093-a8402f10-1baa-4985-a8ab-3ea5a45ae04a.png)
